src/Microsoft.Data/DataFrame.IDataView.cs

```diff
@@ -136,5 +136,86 @@ public override bool MoveNext()
                 return _position < _dataFrame.RowCount;
             }
         }
+
+        public static DataFrame FromIDataView(IDataView dataView)
```

Comment:
We should also add the ability to limit the number of rows, and which columns. See the description in the issue [https://github.com/dotnet/corefxlab/issues/2741](https://github.com/dotnet/corefxlab/issues/2741)

> And we can provide optional arguments to limit the number of rows, and to limit which columns are selected.